decidim-system/config/locales/en.yml

```diff
@@ -64,6 +71,24 @@ en:
         update:
           error: There was a problem updating this admin.
           success: Admin successfully updated.
+      api_user:
+        create:
+          error: API user creation failed.
+          success: API user created successfully.
+        destroy:
+          success: API user successfully deleted.
+        refresh:
+          error: Something went wrong! Please try again later.
+          success: Token refreshed successfully.
+      api_users:
+        index:
+          manage: Manage API users
+          explanation: |
```

Comment:
Let's make this in HTML format and name the key with `_html` suffix which tells the rendering system it is safe HTML.